demos/browser/app/meetingV2/styleV2.scss

```diff
@@ -797,3 +798,38 @@ a.markdown:active {
   outline: none;
   background: none;
 }
+
+.stats-info {
+  position: absolute;
+  background-color: rgba($color: #000000, $alpha: 0.5);
+  color: white;
+  font-size: 10px;
+  bottom: 3rem;
+  left: 0;
+  z-index: $overlay-z-index;
```

Comment:
Do we need this? I see that the video nameplate doesnt specify z-index.

```
.video-tile-nameplate {
  position: absolute;
  bottom: 10px;
  left: 10px;
  color: #fff;
  text-shadow: 0px 0px 5px black;
  letter-spacing: 0.1em;
}
```

Reply:
I am showing the video stats as an overlay over the video tile hence we need a `z-index` for it.